docs/explanation/intro-to-ontologies.md

```diff
@@ -0,0 +1,233 @@
+# Introduction to ontologies
+
+Based on [CL editors training](https://docs.google.com/presentation/d/11WeCHCeGYSPEO7hUYFTdPivptxX4ajj5pVHDm24j4JA) by David Osumi-Sutherland
```

Comment:
A lot of this content in these slides came from content created by me, Melissa Haendel, Matt Brush, and others of the years. I think we need a better way to properly give attribution for all of this content.

Reply:
I think if you acknowledge these folks Matt Brush, Nicole Vasilevsky, Melissa Haendel, Chris Mungall, David Osumi-Sutherland, Matt Yoder, Carlo Torniai, and Simon Jupp - that may get cover it (or close)

Reply:
Oh, I see you have some acknowledgements at the end - definitely include those people too :)